framework/src/utils/SlepcSupport.C

```diff
@@ -96,10 +96,53 @@ getSlepcEigenProblemValidParams()
   params.addParam<unsigned int>("n_eigen_pairs", 1, "The number of eigen pairs");
   params.addParam<unsigned int>("n_basis_vectors", 3, "The dimension of eigen subspaces");
 
+  params.addParam<Real>("e_tol", 1.0e-4, "Relative Tolerance for Eigen Solver");
+  params.addParam<unsigned int>("e_max_its", 10000, "Max Linear Iterations for Eigen Solver");
```

Comment:
Oh, I see, you used `l_max_its` to control power iteration. Then I suggest to changes this to `l_max_power_its` for power iteration and use `l_max_its` for Newton.